src/components/NcActionButtonList/NcActionButtonList.vue

```diff
@@ -0,0 +1,189 @@
+<!--
+  - @copyright Copyright (c) 2023 Ferdinand Thiessen <opensource@fthiessen.de>
+  -
+  - @author Ferdinand Thiessen <opensource@fthiessen.de>
+  -
+  - @license AGPL-3.0-or-later
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<docs>
+This component is made to be used inside of the [NcActions](#NcActions) component slots.
+It allows to show a collapsible list of buttons as an accordion button.
+
+```vue
+	<template>
+		<div style="display: flex; align-items: center;">
+			<NcActions>
+				<NcActionButtonList text="Favorites">
+					<NcActionButton>
+						Foo
+						<template #icon>
+							<Star :size="20" />
+						</template>
+					</NcActionButton>
+					<NcActionButton>
+						Bar
+						<template #icon>
+							<Star :size="20" />
+						</template>
+					</NcActionButton>
+				</NcActionButtonList>
+				<NcActionButtonList name="Books" text="My favorite books">
+					<template #default>
+					<NcActionButton>
+						Lorem ipsum
+						<template #icon>
+							<Book :size="20" />
+						</template>
+					</NcActionButton>
+					<NcActionButton>
+						Dolor sit amet
+						<template #icon>
+							<Book :size="20" />
+						</template>
+					</NcActionButton>
+					</template>
+				</NcActionButtonList>
+			</NcActions>
+		</div>
+	</template>
+	<script>
+	import Book from 'vue-material-design-icons/Book.vue'
+	import Star from 'vue-material-design-icons/Star.vue'
+
+	export default {
+		components: {
+			Book,
+			Star,
+		},
+	}
+	</script>
+```
+</docs>
+
+<template>
+	<NcActionButton ref="listToggle"
+		v-bind="filteredAttributes"
+		:aria-hidden="true"
+		@click="toggleList">
+		{{ text }}
+		<template #content>
+			<ul v-show="contentVisible" :id="contentId">
+				<slot />
+			</ul>
+		</template>
+		<template #icon>
+			<component :is="currentIcon" :size="20" />
+		</template>
+	</NcActionButton>
+</template>
+
+<script>
+import NcActionButton from '../NcActionButton/index.js'
+import IconRight from 'vue-material-design-icons/ChevronRight.vue'
+import IconDown from 'vue-material-design-icons/ChevronDown.vue'
+import GenRandomId from '../../utils/GenRandomId.js'
+
+/**
+ * Button component to be used in Actions
+ */
+export default {
+	name: 'NcActionButtonList',
```

Comment:
Since this works just as good with other `NcAction*` elements, maybe we should give it a more generic name, like `NcAction[Accordion|Expandable|List]` or so.

![Screenshot 2023-05-30 at 21-02-03 Nextcloud Vue Style Guide](https://github.com/nextcloud/nextcloud-vue/assets/2496460/8d7f59e2-3ce4-4c5c-99d4-06b9bc28358b)